src/interfaces/jira.ts

```diff
@@ -118,3 +123,101 @@ export interface JiraDeployment {
 export interface JiraDeploymentData {
 	deployments: JiraDeployment[];
 }
+
+
+interface PullRequests {
```

Comment:
Should probably specify that this is a `JiraPullRequest`.  Don't need to pluralize as it's just a single entity